### ISSUE TITLE
Teleporting out of a fighter because of radiation storm effect (falling through reality) breaks a lot of stuff.

### ISSUE THREAD

**Issue by Panicat** [BeeStation/NSV13#1787]
Fell through reality while flying a fighter, caused by me not going to maints during a rad storm. Kept teleporting around after I fell through reality. Was able to use my hands and throw things as my only way to move around.

Since I was zoomed out, UI was also fucked afterwards, but reconnecting a few times seems to fix that.

## Reproduction:

-Be in a fighter jet (Rapier in my case), have radiation storm effects, specifically the one that makes you "fall through reality" and teleport.
-Fall through reality.
-Can no longer move, can still press shift and hear the boost sound as if I'm in the fighter jet - can also hear fighter engine running even after admin teleports you to station.
-Basically are one with what used to be a fighter, yet things behave as if you are half in and half not in said fighter.

**Comment by covertcorvid** [comment]
Overall probably need to stop_piloting anytime someone who's flying or gunning is forceMoved
